apps/store/modules/page-decorators.js

```diff
@@ -498,37 +498,36 @@ var pageDecorators = {};
                     }
                 }
             }
-        }else{
+        } else {
             sortBy = "createdDate";
             sort = "DES";
             sortHelp = 'Date/Time Created';
         }
-
-        if(sortBy == "overview_name" && sort == "DES"){
+        if(sortBy == "overview_name" && sort == "DES") {
```

Comment:
Thanks @mushthaq33 , This code segment has been refactored and it is no longer used.